hack/local-setup.sh

```diff
@@ -320,9 +317,13 @@ initController ${KIND_CLUSTER_CONTROL_PLANE}
 # 9. Deploy OCM hub
 deployOCMHub ${KIND_CLUSTER_CONTROL_PLANE}
 
+#10. Deploy Redis
+deployRedis ${KIND_CLUSTER_CONTROL_PLANE}
+
+#11. Deploy MetalLb
 deployMetalLB ${KIND_CLUSTER_CONTROL_PLANE} ${metalLBSubnetStart}
 
-#9. Add workload clusters if MGC_WORKLOAD_CLUSTERS_COUNT environment variable is set
+#12. Add workload clusters if MGC_WORKLOAD_CLUSTERS_COUNT environment variable is set
```

Comment:
Small nitpick.
I don't think this needs to be an ordered list of comments.
We don't even output the numbers.
Any reason not to remove them to avoid unnecessary changes or conflicts as steps are modified in future?

Reply:
i have no issue with it at all makes our lives way easier